drivers/include/net/netdev.h

```diff
@@ -257,6 +257,7 @@ typedef enum {
 struct netdev_radio_rx_info {
     int16_t rssi;       /**< RSSI of a received packet in dBm */
     uint8_t lqi;        /**< LQI of a received packet */
+    uint8_t crc_valid;  /**< CRC was valid (when != 0) */
```

Comment:
Do we expect more such bits in the future?
Then it would be better to make this a `flags` field.

Reply:
There was another PR that added the whole `FCS`, but other than that I can't really think of anything else. I could make it a flag just in case though.

Reply:
Did that